docs/markdown/Style-guide.md

```diff
@@ -11,6 +11,12 @@ Meson build files.
 
 Always spaces.
 
+## Naming Variable
+
+The most consistent naming convention is the snack case. Let say you would
```

Comment:
This should probably be "snake case" rather than "snack case".

Reply:
Ok typo is fixed.